doc/README.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to Altair Documentation
+
+Altair documentation is written in [reStructuredText](http://docutils.sourceforge.net/rst.html) and compiled into html pages using [Sphinx](http://www.sphinx-doc.org/en/master/). Contributing to the documentation requires some extra dependencies and we have some conventions and plugins that are used to help navigate the docs and generate great Altair visualizations. 
+
+## Building the Documentation locally
+
+Using a virtual environment, you will need to build the both the requirements for the altair package,  the `docs/requirements.txt`, and install altair locally from the branch you are updating: 
+
+```
+cd altair
+pip install -r requirements.txt
+pip install -e . 
+pip install -r doc/requirements.txt
+```
+In addition, the sphinx documentation builder needs access to Selenium to support generating images of sample visualizations. You may have selenium already installed but if you do not it can also be installed via pip but is not in the requirements file because of its size and dependencies. 
+
+```
+pip install selenium
+```
+
```

Comment:
I tried doing a fresh install and I had to run 
```
pip install recommonmark
```
as well to make things work. Should `recommonmark` be added to `doc/requirements.txt`? @jakevdp

Reply:
I think ``recommonmark`` is part of ``requirements_dev.txt``. We should probably either add ``pip install requirements_dev.txt`` to the list of steps to follow, or add the sphinx-related requirements to ``doc/requirements.txt``